src/Microsoft.DotNet.Tools.New/Template/NuGet.Config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <!--To inherit the global NuGet package sources remove the <clear/> line below -->
+    <clear />
+    <add key="dotnet-core" value="https://www.myget.org/F/dotnet-core/api/v3/index.json" />
```

Comment:
I don't know if there is a planned push but if there is something you guys need pushed then we should sync on getting those dependencies pushed to nuget.org. Is it just NetStandard.Library package (and its closure) that you are needing?